monitoring/metrics.html.md

```diff
@@ -60,7 +60,8 @@ Queries can be sent to the following endpoint:
 https://api.fly.io/prometheus/<org-slug>/
 ```
 
-You'll need to authenticate with a Fly Access Token sent in the [standard](https://www.rfc-editor.org/rfc/rfc6750.html) Bearer Token format (e.g., an HTTP request header `Authorization: FlyV1 <TOKEN>`), and you may only query series scoped to your organizations.
+You’ll need to authenticate with a Fly Access Token. Depending on the token type, the header uses either the Bearer format (`Authorization: Bearer <token>`) or the FlyV1 format (`Authorization: FlyV1 <token>`). You may only query series scoped to your organizations.
```

Comment:
It would be helpful to link to either: the section below 'using tokens" or another docs page that covers Fly Access tokens in more detail (if we have it).